python/repeated.c

```diff
@@ -241,19 +241,12 @@ PyObject* PyUpb_RepeatedContainer_ToList(PyObject* _self) {
   const upb_fielddef* f = PyUpb_RepeatedContainer_GetField(self);
   size_t n = upb_array_size(arr);
   PyObject* list = PyList_New(n);
-  PyObject* ret = list;
-  if (!list) return NULL;
   for (size_t i = 0; i < n; i++) {
     PyObject* val = PyUpb_UpbToPy(upb_array_get(arr, i), f, self->arena);
-    if (!val) {
-      val = Py_None;
-      Py_INCREF(val);
-      ret = NULL;
-    }
+    if (!val) return NULL;
```

Comment:
if you return null here, don't you still need to DECREF(list) ?

Reply:
Done.